source/user-manual/registering/index.rst

```diff
@@ -1,5 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: Learn more about how to register Wazuh agents on Linux, Windows or macOS X in this section of our documentation.
+  
```

Comment:
I can still see the old meta description under the title of this section (lines 12/14) and it needs to be removed.
If you've already fixed it later on, dismiss this comment.